app/components/icons/assets/_Default.tsx

```diff
@@ -1,21 +1,30 @@
-import randomColor from 'randomcolor'
 import * as React from 'react'
-import Svg, { Circle, SvgProps, Text } from 'react-native-svg'
+import randomColor from 'randomcolor'
+import { View, Text } from 'react-native'
+import { tailwind } from '../../../tailwind'
+
+interface _DefaultProps extends React.SVGProps<SVGSVGElement> {
+  testID?: string
+  style?: object
+}
```

Comment:
these default props will never be able to be passed in as on the upstream, only `SVGProps<SVGSVGElement>` is exposed



https://github.com/DeFiCh/wallet/blob/c60d9f5f9a8855a5b1f6b4ecf0ad032f5529c985/app/components/icons/assets/index.ts#L37

Reply:
Thanks for the comment, I will make changes accordingly